{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:14.761Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:14.766Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:14.822Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:14.925Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:15.128Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:15.257Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:15.817Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:16.115Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:16.145Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:16.253Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:16.261Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:16.317Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:17.135Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:17.794Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:18.244Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:22.750Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:22.854Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:23.290Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:23.645Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:17:25.630Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:12.031Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:12.223Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:12.456Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:12.875Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:14.837Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:27.189Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:27.201Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:27.285Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:28.224Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:29.842Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:30.896Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:30.975Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:31.310Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:31.562Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:33.113Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.169Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.172Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.251Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.313Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.527Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:49.725Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:50.485Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:50.648Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:51.166Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:51.393Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:53.768Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:53.799Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:53.899Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:54.409Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:55.565Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:57.520Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:57.719Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:57.879Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:58.634Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:24:59.711Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:08.336Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:08.520Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:08.752Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:08.832Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:10.010Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:17.482Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:17.574Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:18.033Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:18.979Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:25:20.555Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:47.329Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:47.354Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:47.461Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:47.480Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:49.414Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:59.229Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:59.459Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:26:59.810Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:00.034Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:01.045Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:02.273Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:02.293Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:02.670Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:02.967Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:02.999Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:03.662Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:03.681Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:04.117Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:04.667Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:04.678Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:04.874Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:05.182Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:05.721Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:06.567Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:06.667Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:15.012Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:15.126Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:15.512Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:16.068Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:16.184Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:16.681Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:17.086Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:17.275Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:23.156Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:23.268Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:23.653Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:23.980Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:25.861Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:26.688Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:26.943Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:27.325Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:27.750Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:28.414Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:28.552Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:28.630Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:29.073Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:29.703Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:29.927Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:31.902Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:32.002Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:32.399Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:32.952Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:34.632Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:37.347Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:37.384Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:37.417Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:38.353Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:38.555Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:38.905Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:39.003Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:47.253Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:47.435Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:47.592Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:48.358Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:27:48.583Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:28:03.873Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1057:14)","timestamp":"2016-05-26T21:28:09.011Z"}
